dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+                    .append("=$date.toDate(\"yyyy-MM-dd\", \"")
+                    .append(dbFormat)
+                    .append("\"))");
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
+                    .append(shortFormat)
+                    .append("\" )");
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("DBFormat=\"")
+                    .append(dbFormat)
+                    .append("\" )");
+            } else if (field instanceof TimeField) {
+                String shortFormat = "";
+                if (contFieldValueObject != null && contFieldValueObject instanceof Date) {
+                    shortFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "H:mm:ss");
+                }
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
+                    .append(shortFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("= $date.toDate(\"H:mm:ss\", \"")
+                    .append(shortFormat)
+                    .append("\"))");
+            } else if (field instanceof DateTimeField) {
+                String shortFormat = "";
+                String longFormat = "";
+                String dbFormat = "";
+                if (contFieldValueObject != null && contFieldValueObject instanceof Date) {
+                    shortFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "MM/dd/yyyy");
+                    longFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "MM/dd/yyyy H:mm:ss");
+                    dbFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "yyyy-MM-dd H:mm:ss");
+                }
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("= $date.toDate(\"yyyy-MM-dd H:mm:ss\", \"")
+                    .append(dbFormat)
+                    .append("\"))");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
+                    .append(shortFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("LongFormat=\"")
+                    .append(longFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("DBFormat=\"")
+                    .append(dbFormat)
+                    .append("\" )");
+            } // http://jira.dotmarketing.net/browse/DOTCMS-2869
+              // else if (field.getFieldType().equals(Field.FieldType.CUSTOM_FIELD.toString())){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)